docs/bias_correction.rst

```diff
@@ -1,22 +1,84 @@
+
+`Callable workflow`_ : ``apply_annual_blendmorph``, ``apply_interval_blendmorph``
+
+.. _`Callable workflow`: https://bmorph.readthedocs.io/en/develop/api.html#module-bmorph.core.workflows
+
+bmorph Parameters
+-----------------
+
+Conditioning: edcdfm vs mdcdedcdfm
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+EDCDFm
+""""""
+
+Before describing how conditioning is integrated into ``bmorph`` bias correction, we need to discuss the standard method. 
+``bmorph`` implements part of PresRat bias correction from Pierce et al. (2015), which is an extension of Equidistant quantile matching (EDCDFm) technique of Li et al. (2010). ``bmorph`` uses the amended EDCDFm to compute multiplicative changes in the quantiles of a Cumulative Distribution Fuction (CDF). Here, only flow values are used to construct the CDFs. 
+
+.. image:: Figures/Quantile_Map_Diagram.png
+    :alt: Quantile mapping is shown between a Modeled CDF curve (purple) and Observed CDF curve (grey), where x_modeled is inputed to the moodeled curve to get q_modeled, which is compared to Observed at q_modeled to get x_corrected.
+
+mDCDEDCDFm
+""""""""""
+
+multiDimensional ConDitional EquiDistant CDF matching function, (mDCDEDCDFm), incorporates meteorlogic data into the ``bmorph`` bias correction process to condition flow time series to other hydrologically relevant information. By creating a series of CDFs based on meteorlogic data, (such as minimum daily temperature), ``bmorph`` can select a CDF that will not only correct the time series, but most closely match the metrologic conditions simulated.
+    
+.. image:: Figures/conditioning_diagram_with_arrows.png
+    :alt: Multidimensional CDF functions are shown has heat maps with raw taking up left and refernce taking up center. A cumulative proability plot compares raw model CDF and Reference flow CDF for one meterologic slice of the heatmaps left of it. Arrows demonstrate mapping the reference data in the heat maps to the raw model data for bias correction.
+
+.. math::
 
+    \tilde{x_{mp}} = x_{mp} + F^{-1}_{oc}(F_{mp}(x_{mp}|y_{mp})|y_{oc})
+                            - F^{-1}_{mc}(F_{mp}(x_{mp}|y_{mp})|y_{mc})
+                            
+Citations
+---------
 
-Workflows for bmorph
---------------------
+Pierce, D. W., Cayan, D. R., Mauerer, E. P., Abatzoglou J. T., & Hegewisch, K. C. (2015). Improved Bias Correction Techniques for Hydrological Simulations of Climate Change. *Journal of Hydrometeorology, 16*(6), 2421-2442. http://dx.doi.org/10.1175/JHM-D-14-0236.1_
 
+Liang, X., Lettenmaier, D. P., Wood, E. F., & Burges, S. J. (1994). A simple hydrologically based model of land surface water and engergy fluxes for general circulation models. *Journal of Geophysical Research: Atmosphers 99*(D7), 14415-14428. http://dx.doi.org/10.1029/94JD00483_
```

Comment:
This reference doesn't appear to need to be here?

Reply:
You are correct.